test/compiler/parsing/fixtures/watf/instruction/memargs/expected.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "Program",
+  "body": [
+    {
+      "type": "Instr",
+      "id": "load",
+      "object": "i64",
+      "args": [
+        {
+          "type": "Instr",
+          "id": "const",
+          "object": "i32",
+          "args": [
+            {
+              "type": "NumberLiteral",
+              "value": 0
+            }
+          ]
+        }
+      ],
+      "namedArgs": {
+        "offset": "0x000",
+        "align": "1"
```

Comment:
So basically now we can have `key=value` pairs in instruction. This is generic enough and cover the case in #29.

Ideally I would like to have more specific types/objects for some instructions, currently the `namedArgs` bypasses the type checking.